<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiningDialog</class>
 <widget class="QDialog" name="MiningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mining</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../skydoge.qrc">
    <normaloff>:/icons/tx_mined</normaloff>:/icons/tx_mined</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Network info</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelHeight">
        <property name="text">
         <string>Current block height: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelWeight">
        <property name="text">
         <string>Current block weight: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTxns">
        <property name="text">
         <string>Current block txns: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDiff">
        <property name="text">
         <string>Difficulty:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelHashps">
        <property name="text">
         <string>Network hash/s: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPooled">
        <property name="text">
         <string>Pooled txns:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelWarnings">
        <property name="text">
         <string>Warnings:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxAbandonFailedBMM">
     <property name="text">
      <string>Abandon failed BMM requests</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="spinBoxThreads">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="suffix">
        <string> Thread(s)</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStartMining">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Start Mining</string>
       </property>
       <property name="icon">
        <iconset resource="../skydoge.qrc">
         <normaloff>:/icons/tx_mined</normaloff>:/icons/tx_mined</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStopMining">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Stop Mining</string>
       </property>
       <property name="icon">
        <iconset resource="../skydoge.qrc">
         <normaloff>:/icons/quit</normaloff>:/icons/quit</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAddRemove">
       <property name="text">
        <string>Manage Sidechains</string>
       </property>
       <property name="icon">
        <iconset resource="../skydoge.qrc">
         <normaloff>:/icons/options</normaloff>:/icons/options</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWithdrawalVote">
       <property name="text">
        <string>ACK Withdrawals</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelMinerOutput">
     <property name="text">
      <string>Miner output</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameMiningOutput">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="labelHashTarget">
        <property name="font">
         <font>
          <family>Noto Mono</family>
         </font>
        </property>
        <property name="text">
         <string>Target hash: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelHashBest">
        <property name="font">
         <font>
          <family>Noto Mono</family>
         </font>
        </property>
        <property name="text">
         <string>Lowest Hash:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNonce">
        <property name="text">
         <string>Nonce:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../skydoge.qrc"/>
 </resources>
 <connections/>
</ui>
